tests/drivers/test_forecast.py

```diff
@@ -165,3 +166,27 @@ def test_create_directory_structure():
             forecast_obj.create_directory_structure(run_directory, "quit")
         assert pytest_wrapped_e.type == SystemExit
         assert pytest_wrapped_e.value.code == 1
+
+def test_create_forecast_obj():
+    ''' Tests that files fron config_obj are being staged in run directory'''
+
+    with tempfile.TemporaryDirectory() as run_directory:
+
+        test_yaml = os.path.join(uwtools_file_base,pathlib.Path("../fixtures/expt_dir.yaml"))
+        test_cfg = config.YAMLConfig(test_yaml)
+        forecast_obj = FV3Forecast()
+
+        forecast_obj.create_directory_structure(run_directory)
+
+        files_to_stage = test_cfg.get('static', {})
+
+        for target, file_path in files_to_stage.items():
+            pathlib.Path(file_path).touch()
+
+        forecast_obj.stage_fix_files(run_directory, test_cfg)
+
+        for target, file_path in files_to_stage.items():
+            assert os.path.isfile(file_path)
+            assert os.path.isfile(target)
```

Comment:
Is this line checking the current directory for these links instead of the `run_directory`? Maybe the links exist there from an earlier iteration on your system?